docs/DeveloperGuide.md

```diff
@@ -304,6 +304,33 @@ by just looking at their displayed index number without having to refer to an ex
 automatically handle the visual ordering and representation of transactions with the `TransactionsListPanel` in the UI. The details
 and diagrams for this part will be elaborated further in the UI section (and other relevant sections) of this Developer Guide.
 
+#### Edit Transaction
+
+##### Overview
+
+The `edit_transaction` command edits an existing `Transaction` from the `TransactionList` in UniCash.
+
+The following sequence diagram shows how the different components of UniCash interact with each other. It is further explained in the **Details** section below.
+
+<img src="images/unicash/EditTransactionSequenceDiagram.png" width="1200" />
+
+
+##### Details
+
+1. The user specifies the transaction to be edited by first stating the (one-based) index of the transaction they want to edit. This
+   is followed by listing the fields and updated values that they would like to edit.
+   (E.g. `edit_transaction 1 n/Food Clique c/food`)
+2. The raw user input will be parsed by `EditCommandTransactionParser`, and if it is invalid, `ParserException` is thrown, prompting for the user to enter again. It is important to note that at this point, a valid index is any integer that is 1 or greater.
```

Comment:
"raw user input" seems abit odd since it actually only parses the `1 n/Food Clique c/food` part whereas raw input means it includes the `edit_transaction` in front, just a very minor nit